### PR TITLE
Version 2.8.3

### PR DESCRIPTION
**Pull request by corbinlc**

### CODE REVIEW

**Review by Yoosef2020 — APPROVED** [review]

File: app/build.gradle [hunk 1]
```diff
@@ -240,20 +245,20 @@ preBuild.dependsOn checkIfAssetsMissing
...
     // Androidx versions
     def room_version = '2.1.0-beta01'
     def lifecycle_version = '2.2.0-alpha01'
     def support_library_version = '1.1.0-alpha06'
     def preference_version = '1.1.0-alpha05'
     def ktx_version = '1.0.2'
-    def billing_version = '2.2.1'
+    def billing_version = '3.0.3'
 
     implementation project(':terminal-term')
```
Comment (vivox80pro) [inline app/build.gradle:263]:
```suggestion
     implementation project(':terminal-term')
```

Comment (vivox80pro) [reply app/build.gradle:263]: #1444

Comment (vivox80pro) [reply app/build.gradle:263]: #1444

File: app/build.gradle [hunk 2]
```diff
@@ -22,16 +22,21 @@ android {
...
         buildConfigField 'boolean', 'ENABLE_PLAY_SERVICES', 'true'
 
+        //these are used to help determine the resolution
+        buildConfigField 'boolean', 'FORCE_PORTRAIT_GEOMETRY', 'true'
+        buildConfigField 'int', "MAX_DIMENSION", "1280"
+        buildConfigField 'int', "MIN_DIMENSION", "360"
+
         // Ignore Sentry packages that cause lint failure
         lintOptions {
             lintConfig file("$projectDir/src/main/resources/lint.xml")
```
Comment (vivox80pro) [inline app/build.gradle:42]: #1444

**Review by vivox80pro — COMMENTED** [review]
app/src/main/java/tech/ula/viewmodel/MainActivityViewModel.

**Review by moathdarwesh — APPROVED** [review]

**Pull request merged**